cmd/maya-apiserver/spc-watcher/handler.go

```diff
@@ -79,20 +80,17 @@ func (c *Controller) spcEventHandler(operation string, spcGot *apis.StoragePoolC
 		}
 
 		return addEvent, err
-		break
```

Comment:
go has not fallthrough in switch, break not needed. Especially after a return.

Reply:
Good pick!